### ISSUE TITLE
ReFS .....

### ISSUE THREAD

**Issue by gpatator** [Joachim-Otahal/Windows-ShadowCopy#2]
Doesn't see ReFS partition.

**Comment by Joachim-Otahal** [comment]
This is true. Both scripts have "FileSystem -eq "NTFS" at several places, and the config tool even uses a specific NTFS dll call to make them visisble. The same call explorer.exe uses to make local shadowcopies visible after a fresh reboot.
This NTFS .dll call is buggy in Windows 11 since insider June 2022, see here: https://learn.microsoft.com/en-us/answers/questions/935493/windows-11-all-22h2-builds-explorer-exe-previous-v?page=4#answers
The fix will come with the April 2024 update since, by sheer luck, someone from Microsoft saw my constant complaining in public forums. And made it a bug entry and is constantly pushing to geht it fixed. Maybe I will go on with ReFS after that. But you can try too, of course.
